docs/quickstart.md

```diff
@@ -0,0 +1,86 @@
+# AzureTRE Deployment QuickStart
+
+## Deployment
+The AzureTRE uses Terraform infrastructure as code templates that pull down Docker container images to run the web applications.
+
+The most straightforward way to get up and running is to deploy direct from the `microsoft/AzureTRE` repository. Production deployments should take advantage of your chosen DevOps CD tooling.
+
+### Clone the repository
+Clone the repository to to your local machine ( `git clone https://github.com/microsoft/AzureTRE.git` ) or you may choose to use our preconfigured dev container via GitHub Codespaces.
+
+![Clone Options](../docs/assets/clone_options.png)
+
+### Pre-requisites
+
+You will require the following pre requisites installed. They will already be present if using GitHub CodeSpaces, or use our Dev Container in VSCode.
+- Terraform >= v0.15.3. The following instructions use local terraform state, you may want to consider [storing you state remotely in Azure](https://docs.microsoft.com/en-us/azure/developer/terraform/store-state-in-azure-storage)
+- Azure CLI >= 2.21.0
+
+You will also need:
+- An Azure Subscription
+- GitHub user id and [personal access token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token) with scope `packages:read`. This token is used to by the web apps pull the Docker images. This can be any GitHub account, and does not need to be part of the Microsoft GitHub organisation.
```

Comment:
nit: typo
```suggestion
- GitHub user id and [personal access token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token) with scope `packages:read`. This token is used to pull the web apps Docker images from GitHub package repository. This can be any GitHub account, and does not need to be part of the Microsoft GitHub organisation.
```

Reply:
done